### DISCUSSION TITLE
GIF figures are not exported properly to PDF

### DISCUSSION THREAD

**Discussion by RodrigoAVargasHdz** [marp-team/marp#410] [Q&A] [answered]
When I do "Export Slide Deck" to generate the PDF file, I found that the GIF figure is not properly shown in the generated file, meaning no animation only the first frame. However, the VS visualizer does show the animation. 
Is there a specific setting I have to modify?

**Comment by yhatt** [comment, marked as answer]
Probably duplicated with #223.

_Any kinds of animations have not supported in PDF generated from Marp._ If you want to give a presentation with animations, you always have to use HTML.

When you are using Marp for VS Code, choose "HTML slide deck" from the file format in the save dialog, and export to `xxxxxx.html`.

If you often tend to use animations, also can set HTML as the default format from VS Code settings. Refer to [README.md](https://github.com/marp-team/marp-vscode#supported-file-types:~:text=Default%20file%20type%20can%20choose%20by%20markdown.marp.exportType%20preference.).
